lib/LaTeXML/Package/fancybox.sty.ltxml

```diff
@@ -0,0 +1,49 @@
+# -*- mode: Perl -*-
+# /=====================================================================\ #
+# |  fancybox                                                           | #
+# | Implementation for LaTeXML                                          | #
+# |=====================================================================| #
+# | Part of LaTeXML:                                                    | #
+# |  Public domain software, produced as part of work done by the       | #
+# |  United States Government & not subject to copyright in the US.     | #
+# |---------------------------------------------------------------------| #
+# | Bruce Miller <bruce.miller@nist.gov>                        #_#     | #
+# | http://dlmf.nist.gov/LaTeXML/                              (o o)    | #
+# \=========================================================ooo==U==ooo=/ #
+package LaTeXML::Package::Pool;
+use strict;
+use warnings;
+use LaTeXML::Package;
+
+# These could be made to depend on \fboxsep, \fboxrule, \cornersize
+DefMacro('\cornersize OptionalMatch:* {}', '');
+our $fancybox_border = 'border:1px solid black;';
+our $fancybox_shadow = 'border:1px solid black; box-shadow: 5px 5px 10px black;';
+our $fancybox_double = 'border:3px double black;';
+our $fancybox_oval   = $fancybox_border . 'border-radius:5px;';
+our $fancybox_Oval   = 'border:2px solid black;' . 'border-radius:5px;';
```

Comment:
Isn't it more robust to make these into class names, and attach a custom `fancybox.css` to the PR that people can customize/override?

But yeah, maybe that's long-term, maybe a comment that says "TODO: Refactor to classes..." is the best I can ask for in the short-term.